packages/host/src/Host.tsx

```diff
@@ -42,6 +50,122 @@ const Heading: FC<{ locale: string }> = props => (
   </div>
 );
 
+const CustomKBarSearch = styled(KBarSearch)(({ theme }) => ({
+  width: 500,
+  height: 50,
+  fontSize: 20,
+  backgroundColor: theme.palette.primary.main,
+  borderRadius: '5px',
+  ':focus-visible': {
+    outline: 'none',
+  },
```

Comment:
I found the grey text hard to read - what about this?
```suggestion
  color: '#fff',
  width: 500,
  height: 50,
  fontSize: 20,
  backgroundColor: theme.palette.primary.main,
  borderRadius: '5px',
  ':focus-visible': {
    outline: 'none',
  },
  '::placeholder': {
    color: '#fff',
    opacity: 0.75,
  },
```